Cargo.toml

```
@@ -18,19 +18,21 @@ aurora-engine-sdk = { git = "https://github.com/aurora-is-near/aurora-engine.git
 aurora-engine-transactions = { git = "https://github.com/aurora-is-near/aurora-engine.git", tag = "2.7.0" }
 aurora-engine-types = { git = "https://github.com/aurora-is-near/aurora-engine.git", tag = "2.7.0" }
 base64 = "0.13.0"
+bitflags = "1.3"
 borsh = "0.9"
 ethabi = "17.2"
 hex = "0.4"
-near-plugins = { git = "https://github.com/aurora-is-near/near-plugins", tag = "v0.0.1" }
+near-plugins = { git = "https://github.com/aurora-is-near/near-plugins", tag = "v0.0.2" }
 near-contract-standards = "4.0.0"
-near-sdk = "4.0.0"
+near-sdk = "4.1.0"
 near-token-common = { path = "near-token-common" }
 serde = "1"
 serde_json = "1"
 tokio = { version = "1.18", features = ["full"] }
 toml = "0.5"
 uint = { version = "0.9.3", default-features = false }
-workspaces = "0.6"
+# Feature `unstable` is used to enable `compile_project()`.
+workspaces = { version = "0.6", features = ["unstable"] }
```

```suggestion
workspaces = "0.6"
```

Feature `unstable` shouldn't be needed anymore.